plugins/GSdx/GSCrc.cpp

```diff
@@ -251,7 +253,15 @@ CRC::Game CRC::m_games[] =
 	{0x774DE8E2, ValkyrieProfile2, JP, 0},
 	{0x04CCB600, ValkyrieProfile2, EU, 0},
 	{0xB65E141B, ValkyrieProfile2, DE, 0}, // PAL German
-	{0xC70FC973, ValkyrieProfile2, IT, 0}, 
+	{0xC70FC973, ValkyrieProfile2, IT, 0},
+	{0xCE4933D0, RatchetAndClank, US, 0},
+	{0x76F724A3, RatchetAndClank, EU, 0},
+	{0xB3A71D10, RatchetAndClank2, US, 0}, // Going Commando
+	{0x38996035, RatchetAndClank2, US, 0},
+	{0x2F486E6F, RatchetAndClank2, EU, 0},
+	{0x45FE0CC4, RatchetAndClank3, US, 0}, //  Up Your Arsenal
+	{0x9BFBCD42, RatchetAndClank4, US, 0}, // Deadlocked
+	{0x8661F7BA, RatchetAndClank5, US, 0}, // Size Matters
 	{0x47B9B2FD, RadiataStories, US, 0},
```

Comment:
These new CRCs should be in their own section that denotes them from CRCs required for CRC hacks.

```
//Auto Mipmapping CRCs
{0xC70FC973, ValkyrieProfile2, IT, 0},
{0xCE4933D0, RatchetAndClank, US, 0},
...
```

Reply:
I didn't because some games were already added in the list but sure I can move the ones that weren't.